.github/workflows/release.yml

```diff
@@ -0,0 +1,160 @@
+
+      - name: Check signature
+        run: |
+          releasever=${{ github.ref }}
+          releasever="${releasever#refs/tags/}"
+          TAGCHECK=$(git tag -v ${releasever} 2>&1 >/dev/null) ||
+          echo "${TAGCHECK}" | grep -q "error" && {
+              echo "::error::tag ${releasever} is not a signed tag. Failing release process."
+              exit 1
+          } || {
+              echo "Tag ${releasever} is signed."
+              exit 0
+          }
+        working-directory: src/github.com/containerd/containerd
+
+  build:
+    name: Build Release Binaries
+    runs-on: ${{ matrix.os }}
+    needs: [check]
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-2019]
+
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13.11'
+
+      - name: Set env
+        shell: bash
+        run: |
+          releasever=${{ github.ref }}
+          releasever="${releasever#refs/tags/}"
+          echo "::set-env name=RELEASE_VER::${releasever}"
+          echo "::set-env name=GOPATH::${{ github.workspace }}"
+          echo "::add-path::${{ github.workspace }}/bin"
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          repository: containerd/containerd
+          ref: ${{ github.ref }}
+          path: src/github.com/containerd/containerd
+
+      - name: Install Linux dependencies
+        if: startsWith(matrix.os, 'ubuntu')
+        run: |
+          sudo apt-get install -y btrfs-tools libseccomp-dev
+
+      - name: Make
+        shell: bash
+        env:
+          MOS: ${{ matrix.os }}
+          OS: linux
+        run: |
+          make build
+          make binaries
+          [[ "${MOS}" =~ "windows" ]] && {
+              OS=windows
+          }
+          TARFILE="containerd-${RELEASE_VER#v}-${OS}-amd64.tar.gz"
+          tar czf ${TARFILE} bin/
+          sha256sum ${TARFILE} >${TARFILE}.sha256sum
+        working-directory: src/github.com/containerd/containerd
+
+      - name: Save build binaries
+        uses: actions/upload-artifact@v2
+        with:
+          name: containerd-binaries-${{ matrix.os }}
+          path: src/github.com/containerd/containerd/*.tar.gz*
+
+  release:
+    name: Create Release
+    runs-on: ubuntu-18.04
+    timeout-minutes: 10
+    needs: [build]
+
+    steps:
+      - name: Download builds
+        uses: actions/download-artifact@v2
+        with:
+          path: builds
+      - name: Catalog build assets for upload
+        id: catalog
+        run: |
+          _filenum=1
+          for i in "ubuntu-18.04" "windows-2019"; do
+            for i in `ls builds/containerd-binaries-${i}`; do
+              echo "::set-output name=file${_filenum}::${i}"
+              let "_filenum+=1"
+            done
+          done
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
```

Comment:
Is this autofilling in the content?

Reply:
Nope.. the release flow would require you to `git tag -s -m` < pipe in the markdown from the release toml process. Then this action would trigger and actually create the release, build and upload the assets. The text would stay the same as when the tag was created/signed. Optionally we could run the release tool against the toml, but you had suggested not to in the action.

Reply:
I still do at least 2 manual checks after the release tool runs, I check the output against previous tags to make sure there is nothing out of the ordinary with the output, also check authors for duplication. So running locally, doing those checks, creating the tag, then pushing the tag I think is still fine.

Currently the output from the release tool is used exactly as the content for the tag, then that output is also used for the Github release, except the first line is removed and used as the name of the release and the second line is blank and removed.